python/generational/decisions.py

```diff
@@ -191,5 +196,6 @@ def main():
         pygame.display.update()
         clock.tick(15)
 
+
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
```

Comment:
acroding to PEP, py file should be ending with a blank line

Reply:
good to know! 👍